src/containers/restitutionreferral/RestitutionReferral.js

```diff
@@ -203,7 +203,7 @@ const RestitutionReferral = () => {
     ]);
 
     const entityData = processEntityData(formDataForSubmit, entitySetIds, propertyTypeIds);
-    const associations = [
+    const associations :any[][] = [
```

Comment:
Are all of these really necessary?

Reply:
flow was really upset that it wasn't so i just threw them in there.